src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -504,10 +504,12 @@ internal static RemoteSessionNamedPipeServer CreateRemoteSessionNamedPipeServer(
                 securityAttributes);
 
             int lastError = Marshal.GetLastWin32Error();
+#pragma warning disable IDE0031
             if (securityDescHandle != null)
             {
                 securityDescHandle.Value.Free();
             }
+#pragma warning restore IDE0031
```

Comment:
Just learned that `?.` will strip of the Nullable from `GCHandle?`-type, but I've never used this combo so want to confirm. Anything wrong with doing this? @PaulHigin @iSazonov 
```suggestion
            securityDescHandle?.Free();
```

Reply:
Interesting, but that makes sense.  I agree that we should use this change.